pkg/upgrade/handle_upgrade.go

```diff
@@ -65,15 +65,17 @@ func (ctl *Controller) handlePlans(ctx context.Context) error {
 				return objects, status, nil
 			}
 			logrus.Debugf("PLAN GENERATING HANDLER: plan=%s/%s@%s, status=%+v", obj.Namespace, obj.Name, obj.ResourceVersion, status)
-			concurrentNodeNames, err := upgradeplan.SelectConcurrentNodeNames(obj, nodes.Cache())
-			if err != nil {
-				return objects, status, err
-			}
-			logrus.Debugf("concurrentNodeNames = %q", concurrentNodeNames)
-			for _, nodeName := range concurrentNodeNames {
-				objects = append(objects, upgradejob.New(obj, nodeName, ctl.Name))
+			if upgradeapiv1.PlanLatestResolved.IsTrue(obj) {
```

Comment:
Please instead make the `upgradeapiv1.PlanLatestResolved.IsTrue(obj)` check immediately after the `logrus.Debugf()` line above (which follows the nil check) and reverse it such that if `upgradeapiv1.PlanLatestResolved.IsTrue(obj)` evaluates to `false` then we bail out early and return the same values as the nil check.